doc/api/vm.md

```diff
@@ -108,12 +108,12 @@ for (var i = 0; i < 10; ++i) {
   script.runInContext(context);
 }
 
-console.log(util.inspect(sandbox));
+console.log(sandbox);
 
 // { animal: 'cat', count: 12, name: 'kitty' }
 ```
 
-### script.runInNewContext([sandbox][, options])
+### script.runInNewContext(sandbox[, options])
```

Comment:
An hour later I can report back that I am not git-savvy enough. I've have just added another commit to make this change.